tests/InstrumentationEngine.Lib.Tests/InstrumentationEngine.Lib.Tests.vcxproj

```diff
@@ -107,6 +107,7 @@
     </ProjectReference>
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), EnlistmentRoot.marker))\build\Common.targets" />
+  <Import Project="$(EnlistmentRoot)\build\ClrInstrumentationEngine.cpp.props" />
```

Comment:
This is typically done at the top of the project, not after the targets.

Reply:
It's actually at the middle of the project due to some properties that have to be defined before. I replicated the order from InstrumentationEngine.vcxproj